operations/helm/charts/alloy/values.yaml

```diff
@@ -166,10 +171,13 @@ configReloader:
   # -- Override the args passed to the container.
   customArgs: []
   # -- Resource requests and limits to apply to the config reloader container.
-  resources:
-    requests:
-      cpu: "1m"
-      memory: "5Mi"
+  resources: {}
+#    limits:
+#      cpu: "1m"
+#      memory: "5Mi"
+#    requests:
+#      cpu: "1m"
+#      memory: "5Mi"
```

Comment:
IIUC, you are doing this so that you can set `resources` as a whole rather than `resources/request/cpu` and `resources/request/memory` individually? Isn't there still a way to set `resources` as a whole, without having to tweak the cpu and memory  params individually?

Also, wouldn't this break users who rely on the current defaults?